client/src/components/Navbars/SidebarContents.tsx

```diff
@@ -136,6 +136,7 @@ function SidebarContents({ setShowSidebar }) {
         {topics && topics.length > 0 ? (
           topics.map((topic, idx) => (
             <RouterLink
+              key={idx}
```

Comment:
nit: idx is used in some places, i is used in others

Reply:
I'll change them to idx for now